app/gateway/changelog.md

```diff
@@ -9,6 +9,13 @@ Changelog for supported Kong Gateway versions.
 
 For product versions that have reached the end of sunset support, see the [changelog archives](https://legacy-gateway--kongdocs.netlify.app/enterprise/changelog/).
 
+## 3.5.0
+**Release Date** 2023/11/03
```

Comment:
```suggestion
## 3.5.0.0
**Release Date** TBD
```
Setting that as TBD just in case, since we don't want to promise anything for sure ahead of time.